examples/train_simple_navigation.py

```diff
@@ -300,10 +304,39 @@ def _update_reward_dist(self):
             i -= 1
         self._replay_buffer.update_priority(indices, priorities)
 
+    #@tf.function
+    #somehow @tf.function will make it 10 times slower
+    def _tf_learn(self, inputs, actions, rewards, next_inputs, dones, reward_dist,
```

Comment:
It's very strange. I tried putting this into a separate file and using fake date to call it. It's much faster. And using @tf.function is faster than not using it.